src/main/java/com/meilisearch/sdk/TasksHandler.java

```diff
@@ -58,7 +58,7 @@ TasksResults getTasks() throws MeilisearchException {
      * Retrieves all tasks from the client
      *
      * @param param accept by the tasks route
-     * @return List of task instance
+     * @return TasksResults containing a list of task instance
```

Comment:
All the @returns changes on non index related methods should not be in this PR but if you want 🙈